client/src/components/atoms/Dropdown.tsx

```diff
@@ -0,0 +1,39 @@
+import './style/Dropdown.scss';
+import React, { FunctionComponent, useState } from 'react';
+
+interface Props {
+    title: string
+    header?: React.DetailedHTMLProps<React.HTMLAttributes<HTMLSpanElement>, HTMLSpanElement>  
+    body: React.ReactElement
+}
+
+const Dropdown:  FunctionComponent<Props> = (props: Props) => {
+  const [open, setOpen] = useState(false);
+  const toggle = () => setOpen(!open);
+    
+  return (
+      <div>
+        <div className="dd-wrapper">
+            <div
+                tabIndex={0}
+                className="dd-header"
+                role="button"
+                onKeyPress={() => toggle()}
+                onClick={() => toggle()}
+            >
+                <div className="dd-header__title">
+                <p className="dd-header__title--bold">{props.title}</p>
+                </div>
+                <div className="dd-header__action">
+                    {props.header}
+                </div>
+            </div>
+            {open && (
```

Comment:
It seemed pretty fast to me. I'm not too sure how to do this with CSS. I think it should be ok doing with React. @meganniu  thoughts?

Reply:
Try adding more content to the Dropdown body (20-30 elements at least) and see if it still works as expected. If so, leave it. Otherwise, as a generic dropdown, it may not be okay if we later use it.

Reply:
@Anand-Issac yeah it's a little slow to load. you can actually watch the fulfilled requests move to the bottom of the table as it renders

but we're on a time crunch rn, so create a backlog ticket and link this discussion

https://www.notion.so/uwblueprintexecs/34a44be943164278bfbf43de35e4ef7b?v=448f1b24885146a7b264dbf09eb60be1

Reply:
Done